erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -54,7 +55,7 @@ def validate_stock_qty(self):
 				prev_picked_qty = frappe.get_all("Pick List Item", filters={"sales_order_item":item.get("sales_order_item"), "docstatus":1}, fields=['sum(qty) as prev_picked_qty'])
 
 				if prev_picked_qty:
-					prev_picked_qty = prev_picked_qty[0].prev_picked_qty
+					prev_picked_qty = flt(prev_picked_qty[0].prev_picked_qty)
 					pick_list_qty = ordered_item_qty - prev_picked_qty
```

Comment:
Actually, on second thoughts, for everyone's sanity, just do:

```suggestion
					prev_picked_qty = prev_picked_qty[0].prev_picked_qty
					pick_list_qty = flt(ordered_item_qty) - flt(prev_picked_qty)
```